website/content/en/guides/level-up/e2e-ack-guide.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ensure data is always successfully delivered
+short: End-to-end acknowledgement
+description: Learn how to use end-to-end acknowledgement
+author_github: https://github.com/barieom
+domain: delivery
+weight: 7
+tags: ["delivery", "logs", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand the [basic Vector concepts][concepts]
+* You understand [how to set up a basic pipeline][pipeline]
+
+[concepts]: /docs/about/concepts
+[pipeline]: /docs/setup/quickstart
+{{< /requirement >}}
+
+
+Vector is often deployed in mission critical environments where loss of data is
+non-negotiable. Ergo, assurance that every data flowing through your Vector
+pipeline is successfully delivered to the end destination is critical.
+
+We're excited to walk you through exactly you can accomplish this by using
+Vector's end-to-end acknowledgement feature.
+
+## Getting Started
+
+You can set and control the acknowledgement feature either at the global level or
+the sink level. Let's start with the global configuration option first, as it is
+easy as flipping on a switch. As you would expect from a global configuration, the
+configuration below turns end-to-end acknowledgement on for every source:
+
+```toml
+acknowledgement = true
```

Comment:
Note, `acknowledgement` should be `acknowledgements` everywhere, and the configuration has an `enabled` field. Setting just `acknowledgements = true` works through a compatibility hack.
```suggestion
acknowledgements.enabled = true
```